examples/aggregator/README.md

```diff
@@ -53,3 +53,35 @@ Build and run the Aggregator with the following command:
 
 Aggregator code is in `common` and `module` directories (where `common` defines
 a generic Aggregator data structure).
+
+## Deployment
+
+The Aggregator example contains a `gcp/pod.yaml` file which is a config for
+Google Cloud deployment.
+
+An Aggregator deployment consists of a single pod with two running containers
+(an Aggregator Oak application and a Backend server) and a service with a TCP
+Load Balancer and an assigned static IP `35.246.87.178` that listens on the port
```

Comment:
I would remove the actual IP from here, otherwise it needs to be kept in sync when we regenerate it.